Fix installation commands

The installation commands did not match the location of the source
files within the Orbit source tree.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -11,12 +11,9 @@
 	mkdir -p $(LUA_DIR)
 	cp src/orbit.lua $(LUA_DIR)
 	mkdir -p $(LUA_DIR)/orbit
-	cp src/model.lua $(LUA_DIR)/orbit
-	cp src/cache.lua $(LUA_DIR)/orbit
-	cp src/pages.lua $(LUA_DIR)/orbit
-	cp src/ophandler.lua $(LUA_DIR)/orbit
+	cp src/orbit/*.lua $(LUA_DIR)/orbit
 	mkdir -p $(BIN_DIR)
-	cp src/orbit $(BIN_DIR)
+	cp src/launchers/* $(BIN_DIR)
 	if [ -f ./wsapi/Makefile ]; then \
 	  cd wsapi && make install; \
 	fi
